backend/models/office_hours/oh_event_status.py

```diff
@@ -0,0 +1,26 @@
+"""Represent the status of an office hours event."""
+
+from pydantic import BaseModel
+from typing import Sequence
+
+from oh_ticket import OfficeHoursTicket
+
+__authors__ = ["Sadie Amato, Bailey DeSouza, Meghan Sun, Maddy Andrews"]
+__copyright__ = "Copyright 2024"
+__license__ = "MIT"
+
+
+class TAOfficeHoursEventStatus(BaseModel):
+    """Pydantic model to represent the status of a given event, including tickets in the queue, for a given TA."""
+
+    active_tickets: Sequence[OfficeHoursTicket]
+    num_open: int
+    num_pending: int
```

Comment:
Can we iterate on names of these fields? I'd like to avoid abbreviations, if possible. Some alternative ideas:

`open`
`open_count`
`open_tickets_count`

Additionally, thoughts on `queued` versus `pending`?  The same language should propagate to `ticket_state`. There, `pending` is the only `ing` state. 

Going for a super explicit name here has my vote, I think.